### ISSUE TITLE
"Soften" contributor requirements like unit tests

### ISSUE THREAD

**Issue by mfisher87** [nsidc/earthaccess#525]
Some contributing requirements may be intimidating users from contributing. Let's change our language in the contributing guide to indicate:

If you're not comfortable writing unit tests, we still welcome your contribution! Just please ask for help and someone from the community will do their best to work with you.

**Comment by doug-newman-nasa** [comment]
As long as we don't merge any bug fixes/features _without_ tests I'm OK with that.

**Comment by jhkennedy** [comment]
Yeah, I like saying it's okay to let PR CI/CD run the unit tests instead of needing to do it locally and it's okay to ask for help to make them pass

**Comment by mfisher87** [comment]
>As long as we don't merge any bug fixes/features without tests I'm OK with that.

:100:

**Comment by mfisher87** [comment]
I think a PR template is a good way to communicate the exact checklist in an accessible way, including the "ask for help" part. Then we can shorten the contributing doc a bit by linking to the PR checklist.
